CONTRIBUTING.md

```diff
@@ -0,0 +1,169 @@
+# Contributing to CopilotKit
+
+⭐ Thank you for your interest in contributing!!
+
+Here’s how you can contribute to this repository
+
+## How can I contribute?
+
```

Comment:
I do think you have answered this in later section then why redirecting to discord here.
Instead can add this as faq and place this in later section